tools/automation/vendoring/__init__.py

```diff
@@ -128,6 +128,7 @@ def vendor(ctx, vendor_dir):
     )
     remove_all(vendor_dir.glob('*.dist-info'))
     remove_all(vendor_dir.glob('*.egg-info'))
+    remove_all(vendor_dir.glob('*/py.typed'))
```

Comment:
I'd prefer to keep this in a separate commit so that this isn't coupled with updating vendoring.

You probably thought of the same and might fix this when updating the vendored version -- I just wanted to flag that this is a thing we'd want to change before merging.